packages/graphql-api/src/graphql/mutations.ts

```diff
@@ -1,11 +1,16 @@
 export const mutations = `
-  loginWithPassword(user: String, userFields: UserInput, password: String!): LoginReturn
+  impersonate(accessToken: String! username: String!): ImpersonateReturn
```

Comment:
Can you please change the username with the `LoginUserIdentity` from the `@accounts/types` package?
```

Reply:
@pradel I see rest-express is doing only accessToken and username, should we still change?

https://github.com/accounts-js/accounts/blob/master/packages/rest-express/src/endpoints/impersonate.ts#L12

Reply:
Okay so this looks like an issue on the rest package actually 🤔. the accounts server accept only an object not a string see https://github.com/accounts-js/accounts/blob/master/packages/server/src/accounts-server.ts#L164.
I will open an issue to track it for the rest package.

Reply:
@pradel OK, so for this PR, we should just do as accounts-server.ts requires.